lytical/roster.json

```diff
@@ -76,7 +76,8 @@
       "github": "rwclutterbuck"
       "materials": [  
         {
-          "type": "cv"
+          "type": "cv",
+          {"type": "portfolio", "url": "https://github.com/rwclutterbuck/awesome-projects"}
```

Comment:
Be careful, you have added too much to the first object.
Your code should look like this
        {
          "type": "cv"
        },
        {
            "type": "portfolio",
            "url": "add_your_url_here"
        }
        
        Also if you are going to go the GitHub route, we will need a GitHub page instead of the repo :)